ext/mysqli/tests/mysqli_result_references.phpt

```diff
@@ -84,7 +84,7 @@ opcache.enable=0
     require_once 'clean_table.inc';
 ?>
 --EXPECTF--
-array(7) refcount(2){
+array(7) packed refcount(2){
```

Comment:
Just as an idea which could be also used in regular var_dump()

```suggestion
list(7) refcount(2){
```

Reply:
I had thought about something like this, but decided that it would be confusing because the type remains IS_ARRAY at all times.